esrally/mechanic/telemetry.py

```diff
@@ -443,14 +443,19 @@ def record_stats_per_index(self, name, stats):
 
 
 class NodeStats(TelemetryDevice):
+    """
+    Gathers different node stats.
+    """
+
     internal = False
     command = "node-stats"
     human_name = "Node Stats"
     help = "Regularly samples node stats"
-
-    """
-    Gathers different node stats.
+    warning = """You have enabled the node-stats telemetry device, but requests to the _nodes/stats Elasticsearch endpoint
+          trigger additional refreshes and WILL SKEW results.
+          If you still require this, consider reducing the interval.
```

Comment:
This implies that a reduced interval might be ok but it still might skew results.

Reply:
Agreed. Less is more in this case, will remove.

Reply:
Addressed in https://github.com/elastic/rally/pull/627/commits/698621aaea06c113bc8015a14f505d678f8c3669